py/test/unit/selenium/webdriver/remote/remote_connection_tests.py

```diff
@@ -239,3 +262,52 @@ def mock_no_proxy_settings(monkeypatch):
     monkeypatch.setenv("http_proxy", http_proxy)
     monkeypatch.setenv("no_proxy", "65.253.214.253,localhost,127.0.0.1,*zyz.xx,::1")
     monkeypatch.setenv("NO_PROXY", "65.253.214.253,localhost,127.0.0.1,*zyz.xx,::1,127.0.0.0/8")
+
+
+@patch("selenium.webdriver.remote.remote_connection.RemoteConnection.get_remote_connection_headers")
+def test_override_user_agent_in_headers(mock_get_remote_connection_headers, remote_connection):
+    RemoteConnection.user_agent = "rspec/1.0 (python 3.8)"
```

Comment:
Not that this really matters but guessing this is a copy/pasta from Ruby?

Reply:
Yes, mostly!